src/ontology/hp.Makefile

```diff
@@ -636,3 +636,77 @@ $(TMPDIR)/hp-%-merged.owl: hp-base.owl tmp/%.owl
 
 mappings: 
 	$(MAKE_FAST) ../mappings/hp-snomed.lexmatch.sssom.tsv
+
+.PHONY: diff
+diff: diff-release-base diff-main-branch-base
+
+tmp/hp-edit-pr.obo: hp-edit.owl
+	$(ROBOT) convert --input $< --check false -f obo -o $@
+
+tmp/hp-main-branch.owl:
+	wget https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/src/ontology/hp-edit.owl -O $@
+
+tmp/hp-main-branch.obo: tmp/hp-main-branch.owl
+	$(ROBOT) convert --input $< --check false -f obo -o $@
+
+
+tmp/%.db: tmp/%.owl
+	semsql make $@
+
+tmp/hp-released.owl:
+	wget http://purl.obolibrary.org/obo/hp.owl -O $@
+
+tmp/hp-released.obo: tmp/hp-edit-pr.obo
+	wget http://purl.obolibrary.org/obo/hp.obo -O tmp/hp-released.obo
+
+
+
+kgcl-diff-md-release-base: reports/difference_release_base.md
+kgcl-diff-table-release-base: reports/difference_release_base.tsv
+kgcl-diff-txt-release-base: reports/difference_release_base.txt
+kgcl-diff-yaml-release-base: reports/difference_release_base.yaml
+diff-release-base: kgcl-diff-md-release-base kgcl-diff-table-release-base kgcl-diff-txt-release-base kgcl-diff-yaml-release-base
```

Comment:
All the above are missing `.PHONY` declarations

Reply:
Done